tests/features/test_search_index.py

```diff
@@ -201,6 +204,56 @@ def test_search(self):
         # parameter and check that the number of returned hits is equal to 2
         result = self.index.search('algolia')
         self.assertEqual(result['nbHits'], 2)
+        self.assertEqual(SearchIndex.get_object_position(
+            result, 'nicolas-dessaigne'), 0
+        )
+        self.assertEqual(SearchIndex.get_object_position(
+            result, 'julien-lemoine'), 1
+        )
+
+        self.assertEqual(SearchIndex.get_object_position(result, ''), -1)
+
+        # Call find_object with the following parameters and check that
+        # no object is found
+        with self.assertRaises(ObjectNotFoundException):
+            self.index.find_object(lambda _: False)
+
+        # Call find_object with the following parameters and check that
+        # the first object is returned with a `position=0` and `page=0`
+        found = self.index.find_object(lambda _: True)
+        self.assertEqual(found['position'], 0)
+        self.assertEqual(found['page'], 0)
+
+        def callback(obj):
+            # type: (dict) -> bool
+            return obj.get('company') == 'Apple'
+
+        # Call find_object with the following parameters and check that
+        # no object is found
+        with self.assertRaises(ObjectNotFoundException):
+            self.index.find_object(callback, {
+                'query': 'algolia'
+            })
+
+        # Call find_object with the following parameters and check that
+        # no object is found
+        with self.assertRaises(ObjectNotFoundException):
+            self.index.find_object(callback, {
+                'query': 'algolia',
```

Comment:
```suggestion
                'query': '',
```

because here you want to test the pagination but by querying `algolia` it will always be false.
See https://github.com/algolia/algoliasearch-client-ruby/blob/b15be640fc8455f5e5e5248248d9926400c9ad63/spec/client_spec.rb#L485

Reply:
We need to update the CTS for that. On the CTS is `Algolia`.